server/portal/libs/agave/operations.py

```diff
@@ -99,7 +99,7 @@ def search(client, system, path, offset=0, limit=100, query_string='', **kwargs)
     """
     ngram_query = Q("query_string", query=query_string,
                     fields=["name"],
-                    minimum_should_match='80%',
+                    minimum_should_match='100%',
```

Comment:
Doesn't this remove fuzzy search? i.e. to account for typos

Reply:
Ah my bad, I see it does and that's the intent of the task. I wonder what the implications of removing fuzzy search are

Reply:
If we want to support fuzzy search there are ways to implement it that are supported by ES for that specific purpose

> fuzziness
> (Optional, string) Maximum edit distance allowed for matching. See Fuzziness for valid values and more information.
> fuzzy_max_expansions
> (Optional, integer) Maximum number of terms to which the query expands for fuzzy matching. Defaults to 50.
> fuzzy_prefix_length
> (Optional, integer) Number of beginning characters left unchanged for fuzzy matching. Defaults to 0.
> fuzzy_transpositions
> (Optional, Boolean) If true, edits for fuzzy matching include transpositions of two adjacent characters (ab → ba). Defaults to true.
> lenient
> (Optional, Boolean) If true, format-based errors, such as providing a text value for a numeric field, are ignored. Defaults to false

[https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-query-string-query.html)

[https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-fuzzy-query.html](https://www.elastic.co/guide/en/elasticsearch/reference/current/query-dsl-fuzzy-query.html)

Reply:
Nice - so perhaps we can determine desired fuzziness in a follow up task?